### PR TITLE
script for auto topic nrs

### PR DESCRIPTION
**Pull request by ryabhmd**

### CODE REVIEW

File: src/application/Bertopic/run_bertopic.py [hunk 1]
```diff
@@ -3,40 +3,37 @@
...
-        topic_model = BERTopic(verbose=True)
-        topics, probs = topic_model.fit_transform(docs)
-        topic_model.reduce_topics(docs, nr_topics=nr)
-        topic_model.get_topic_info().to_csv(
-            f"../../../data/results/csv/{label}s_{nr}_topics.csv", index=False
-        )
+    topic_model = BERTopic(verbose=True, nr_topics="auto")
+    topics, probs = topic_model.fit_transform(docs)
+    topic_model.get_topic_info().to_csv(
+        f"../../../data/results/csv/{label}s_{nr}_topics.csv", index=False
```
Comment (esborisova) [inline src/application/Bertopic/run_bertopic.py:14]: nr is not defined. But of course we want to keep track on the number of topics generated by the model and to save this info in files names:

`topics, probs = topic_model.fit_transform(docs)
  num_topics = len(topics)
  topic_model.get_topic_info().to_csv(
         f"../../../data/results/csv/{label}s_{num_topics}_topics.csv", index=False`

File: src/application/Bertopic/run_bertopic.py [hunk 2]
```diff
@@ -3,40 +3,37 @@
...
-        intertopic_map.write_image(
-            f"../../../data/results/pdf/{label}s_intertopic_map_{nr}_topics.html"
-        )
+    intertopic_map = topic_model.visualize_topics()
+    intertopic_map.write_html(
+        f"../../../data/results/html/{label}s_intertopic_map.html"
+    )
+    intertopic_map.write_image(
+        f"../../../data/results/pdf/{label}s_intertopic_map.html"
+    )
```
Comment (esborisova) [inline src/application/Bertopic/run_bertopic.py:28]: 1. add {num_topics} to paths.
2. intertopic_map.write_image() does not accept html, should be .pdf

**Review by esborisova — CHANGES_REQUESTED** [review]
1. fix nr variable
2. fix doc extension 
3. delete data
4. delete folder with virtual env
